vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+        "name": "vNetCopy",
+        "count": 2
+      },
+      "properties": {
+        "addressSpace": {
+          "addressPrefixes": [
+            "[variables('addressPrefix')]"
+          ]
+        },
+        "subnets": [
+          {
+            "name": "[variables('subnetName')]",
+            "properties": {
+              "addressPrefix": "[variables('subnetPrefix')]"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "type": "Microsoft.Network/networkInterfaces",
+      "name": "[concat(variables('nicName'), '1')]",
+      "location": "[variables('location')[0]]",
+      "dependsOn": [
+        "[concat(variables('publicIPAddressName'), '1')]",
+        "[concat(variables('virtualNetworkName'), '1')]"
+      ],
+      "properties": {
+        "ipConfigurations": [
+          {
+            "name": "ipconfig1",
+            "properties": {
+              "privateIPAllocationMethod": "Dynamic",
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses',concat(variables('publicIPAddressName'), '1'))]"
+              },
+              "subnet": {
+                "id": "[variables('subnetRef')[0]]"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[variables('vmName')[0]]",
+      "location": "[variables('location')[0]]",
+      "dependsOn": [
+        "[concat(variables('vhdStorageAccountName'), '1')]",
+        "[concat(variables('nicName'), '1')]"
+      ],
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "[parameters('probeVmSize')]"
+        },
+        "osProfile": {
+          "computerName": "[variables('vmName')[0]]",
+          "adminUsername": "[parameters('adminUsername')]",
+          "adminPassword": "[parameters('adminPassword')]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "[variables('imagePublisher')[0]]",
+            "offer": "[variables('imageOffer')[0]]",
+            "sku": "[variables('windowsOSVersion')[0]]",
+            "version": "latest"
+          },
+          "osDisk": {
+            "name": "osdisk",
+            "vhd": {
+              "uri": "[concat(reference(concat(variables('vhdStorageAccountName'),'1'), providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]).primaryEndpoints.blob, variables('vmStorageAccountContainerName'),'/',variables('OSDiskName'),'.vhd')]"
+            },
+            "caching": "ReadWrite",
+            "createOption": "FromImage"
+          }
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(variables('nicName'), '1'))]"
+            }
+          ]
+        }
+      },
+      "resources": [
+        {
+          "type": "extensions",
```

Comment:
Getting: "Handler 'Microsoft.Azure.Extensions.CustomScript' has reported failure for VM Extension 'CustomScriptExtension' with terminal error code '1007' and error message: 'Install failed for plugin (name: Microsoft.Azure.Extensions.CustomScript, version 2.0.1) with exception The specified executable is not a valid application for this OS platform.'"
Can you share an example for Windows?